apps/verticalhome/js/pinned_apps_navigation.js

```diff
@@ -49,38 +50,46 @@
     }else{
       this.selector = '';
     }
-  }
 
-  PinnedAppsNavigation.prototype = {
+    /**
+     * This is a variable that used for storing personalize mode status
+     * @type {Boolean}
+     */
+    this.personalizeMode = false;
 
     /**
-     * This function resets Main homscreen's state to default one (with clock)
-     * @return {[nothing]} [nothing]
+     * This is a variable that used for for storing rearrange mode status
```

Comment:
Two words 'for' :smile: 
It would be more grammatically if you write 
"This variable stores status of rearrange mode."